index.js

```diff
@@ -6,13 +6,19 @@ if(!command_string) {
     throw new Error("SAF CLI Command String argument is required.");
 }
 
-const allowable_commands = ['convert', 'generate', 'harden', 'scan', 'validate', 'view'];
-if(!allowable_commands.includes(command_string.split(':')[0])) {
-    throw new Error("The command string did not include one of the allowable commands: " + allowable_commands.join(', ') + ". Please reference the documentation for more details.");
+const saf_command = command_string.split(' ');
+
+const allowable_topics = ['convert', 'generate', 'harden', 'scan', 'validate', 'view'];
+const topic = saf_command[0].split(':')[0] || saf_command[0];
```

Comment:
This should check for a colon in saf_command[0] before splitting, or else this will always return the first value:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/66680985/160844796-f44ba1b7-c1d8-44eb-9705-4923763edf38.png">